### ISSUE TITLE
Improve Copy / UX on Account View for Desktop Applications

### ISSUE THREAD

**Issue by dternyak** [MyCryptoHQ/MyCrypto#1621]
A large portion of the space on the default view of the MyCrypto Desktop applications goes to showing disabled wallet formats.

- [ ] Determine a better UX that displays valid formats while making better use of space.

**Comment by jspence425** [comment]
Highlighting "Web3" on the Desktop app displays "This wallet is not supported in the MyCrypto app"
Highlighting "Ledger" on the Desktop app displays " This wallet is only supported at MyCrypto.com"

I understand there's a difference between the two but the messaging either needs to be the same for both (a large majority of our users do not care for the specific reason why they can't access in these manners, they'll just care that it can or cannot be used) or different for both but explained in detail for why you cannot access using these routes.

I would recommend saying "This wallet type is not supported on the MyCrypto app" as a catch-all.

**Comment by wbobeirne** [comment]
I think the goal by the time we launch this is to have these two sorted out. #921 should allow for Ledger on the app, and we're also eyeing https://github.com/MetaMask/mascara which could replace the Web3 option.

**Comment by dternyak** [comment]
Mascara is really early, so unfortunately it's not a good option for the
foreseeable future.

We can look into #921, but I don't see that viable in the next month, but
perhaps a separate ledger implementation is.

**Comment by wbobeirne** [comment]
I think I'd sooner whip up a temporary Ledger implementation that's not #921, than ship without supporting it. We should probably get that on the roadmap.

**Comment by dternyak** [comment]
@wbobeirne Yeah, absolutely. I think the node-hid approach might be the way to go for desktop apps, so we'll just have a separate implementation for desktop.
